site/zh-cn/guides/border-router/get-started.md

```diff
@@ -0,0 +1,75 @@
+# 开始使用 OTBR
+
+Contributor: https://github.com/dsyx
```

Comment:
Usually Contributor is for original content.

For this, we can have a different byline of "Translator: https://github.com/dsyx"

If you can make that update on both files here, @dsyx, that would be great, otherwise we can change it later when we import the translation into openthread.io